R/multigene_udr.R

```diff
@@ -69,7 +69,7 @@ multigene_udr <- function(combined_susie_list, coverage, independent_variant_lis
       mash_input,
       bhat = paste0(cond, ".b"),
       sbhat = paste0(cond, ".s"),
-      z = TRUE,
+      btoz = TRUE,
```

Comment:
filter_invalid_summary_stat() is also called here, so I changed the commands here accordingly